src/main/java/org/radarcns/management/config/OAuth2ServerConfiguration.java

```diff
@@ -201,7 +201,7 @@ public TokenEnhancer tokenEnhancer() {
 
         @Bean
         public TokenStore tokenStore() {
-            return new ManagementPortalJwtTokenStore(accessTokenConverter(), approvalStore());
```

Comment:
Would this `approvalStore()` not allow the MP store to invalidate refresh tokens?

Reply:
I was trying to stick to exactly what the Spring Implementation was doing. There the JwtTokenStore is not actually a "Store" per say, they do not use it to store or invalidate a token using that. The actual approval store in action is the JdbcApprovalStore or the PostGreApprovalStore. However, it is not used in JwtTokenStore based on the existing implementation.